packages/react/src/input/Input.tsx

```diff
@@ -43,7 +51,34 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(
         data-invalid={isInvalid}
         {...sprinkleProps}
       >
-        <input placeholder={placeholder} ref={ref} type={type} {...restProps} />
+        {leftSection && (
+          <Box flex="none" mr="8">
+            {leftSection}
+          </Box>
+        )}
+        <Box
+          asChild
+          className={clsx(
+            styles.input({
+              variant,
+            }),
+          )}
```

Comment:
let's shorten this to a single line
```suggestion
          className={styles.input({ variant })}
```

and remove `clsx` since we're not merging any classes here

Reply:
Okay vai